jsonpatcher/jsonpatcher_test.go

```diff
@@ -35,18 +44,37 @@ func TestJsonPatcher_Migration(t *testing.T) {
 	if !out.time().Equal(time.Time{}) {
 		t.Error("non-encodable time should have zero-value")
 	}
+	b.Reset()
+	e = gob.NewEncoder(&b)
+	if err := e.Encode(out); err != nil {
+		t.Errorf("failed to gob encode event with empty time.Time timestamp: %v", err)
+	}
 
 	ts := time.Now()
 	in2 := patchEvent{Timestamp: ts.UnixNano(), ID: "123", CollectionName: "abc"}
 	node2, err := cbornode.WrapObject(in2, mh.SHA2_256, -1)
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	b.Reset()
+	e = gob.NewEncoder(&b)
+	if err := e.Encode(in2); err != nil {
+		t.Errorf("failed to gob encode event with int64 timestamp: %v", err)
+	}
+
 	err = cbornode.DecodeInto(node2.RawData(), &out)
 	if err != nil {
 		t.Errorf("failed to decode event with int64 timestamp: %v", err)
 	}
 	tsout := out.time()
+
+	b.Reset()
+	e = gob.NewEncoder(&b)
+	if err := e.Encode(tsout); err != nil {
```

Comment:
Did you mean for this to be `e.Encode(out)` as above?

Reply:
yeap, fixed